libtsuba/src/RDG.cpp

```diff
@@ -268,15 +274,30 @@ tsuba::RDG::WritePartArrays(const katana::Uri& dir, tsuba::WriteGroup* desc) {
     });
   }
 
-  if (host_to_owned_global_ids_ != nullptr) {
-    auto h2o_res = StoreArrowArrayAtName(
-        host_to_owned_global_ids_, dir, kHostToOwnedGlobalIDsPropName, desc);
-    if (!h2o_res) {
-      return h2o_res.error();
+  if (host_to_owned_global_node_ids_ != nullptr) {
+    auto h2on_res = StoreArrowArrayAtName(
+        host_to_owned_global_node_ids_, dir, kHostToOwnedGlobalNodeIDsPropName,
+        desc);
+    if (!h2on_res) {
+      return h2on_res.error();
+    }
+    next_properties.emplace_back(tsuba::PropStorageInfo{
+        .name = kHostToOwnedGlobalNodeIDsPropName,
+        .path = std::move(h2on_res.value()),
+        .persist = true,
```

Comment:
real nit, but `h2on `and `h2oe `look an awful lot alike.

Reply:
Yeah, I'll try to come up with a better name.